qpc/report/aggregate.py

```diff
@@ -44,11 +48,24 @@ def __init__(self, subparsers):
             metavar="REPORT_ID",
             help=_(messages.REPORT_REPORT_ID_HELP),
         )
+        self.parser.add_argument(
+            "--output-file",
+            dest="path",
+            metavar="PATH",
+            help=_(messages.REPORT_PATH_HELP),
+        )
         self.report_id = None
 
     def _validate_args(self):
         CliCommand._validate_args(self)
 
+        try:
+            if (args_path := getattr(self.args, "path", None)) is not None:
```

Comment:
[details are doing `if self.args.path`](https://github.com/quipucords/qpc/blob/main/qpc/report/details.py#L91). That has two consequences:

- they need to set `path` in `Namespace()` in all unit tests
- if `self.args.path` is empty string, it will only bail out after making a request, in `write_file` call

I decided I don't want to modify existing unit tests for aggregate (after all, `--output-file` is supposed to be optional), and I'd rather catch empty value early than during `Path().open()` call.